components/ingest-service/backend/elastic/mappings/node_state.go

```diff
@@ -65,7 +65,13 @@ var nodeProps = `
 		"type": "keyword"
 	},
 	"status": {
-		"type": "keyword"
+		"type": "keyword",
+		"fields": {
+			"engram" : {
+				"type": "text",
+				"analyzer": "autocomplete"
+			}
+		}
```

Comment:
Allowing the status to have autocomplete. Without this, the suggestions do not work.